corehq/apps/hqwebapp/static/cloudcare/scss/formplayer-common/module.scss

```diff
@@ -299,7 +299,7 @@ button.clickable-icon {
   }
 }
 
-@media(max-width: $screen-sm-max) {
+@include media-breakpoint-down(lg) {
```

Comment:
I expected this, and the other `down` lines, to change to `md` since the `media-breakpoint-up` ones change by one size, but I chose this by doing some testing and comparing with the styles on staging, didn't think too hard about it.

Reply:
Noting the numbered classes change a size ([here](https://github.com/dimagi/commcare-hq/blob/81c2b6ab65890e664d105da20ea4a64baf3465ba/corehq/apps/hqwebapp/utils/bootstrap/spec/bootstrap_3_to_5.json#L59-L68)) but other size-based classes do not ([see here](https://github.com/dimagi/commcare-hq/blob/81c2b6ab65890e664d105da20ea4a64baf3465ba/corehq/apps/hqwebapp/utils/bootstrap/spec/bootstrap_3_to_5.json#L42-L48)) so 🤷